target/step159-2020-1/script.js

```diff
@@ -0,0 +1,8 @@
+function makeGraph() {
+  fetch('/graph')
+    .then((response) => response.json())
+    .then((data) => {
+      dataObject = JSON.parse(data);
+      console.log(dataObject);
```

Comment:
Why are there two hours.csv, difficulty.csv, and script.js?

Reply:
Could you clarify what you mean here. On my end I only one of each file.

Reply:
Resolved think it was I got confused :)

Reply:
I'm seeing the same two hours.csv, difficulty.csv, and script.js